### PR TITLE
fix nil deference

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: cmd/runtimetest/main.go [hunk 1]
```diff
@@ -178,7 +182,7 @@ func validateCapabilities(spec *rspec.Spec) error {
 	expectedCaps3 := make(map[string]bool)
 	expectedCaps4 := make(map[string]bool)
 	expectedCaps5 := make(map[string]bool)
-	if spec.Process.Capabilities != nil {
+	if spec.Process != nil && spec.Process.Capabilities != nil {
```
Comment (wking) [inline cmd/runtimetest/main.go]: I'd rather move this guard to the top of this function, and have:

```go
if spec.Process == nil || spec.Process.Capabilities == nil {
    return nil
}
```

I don't see the point to initializing `processCaps`, etc. when we aren't going to use them.

Comment (q384566678) [reply cmd/runtimetest/main.go]: updated, thanks.

**Comment by Mashimiao** [comment]
LGTM

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**